.github/workflows/mirror-images.yml

```diff
@@ -12,9 +12,19 @@ on:
         type: string
         required: true
         default: "1.0.0"
+  workflow_run:
+    workflows: ['Create Production Tag']
+    types: [completed]
+#    branches: [main]
```

Comment:
This will become restricted to `main`; commented out for the time being for testing purposes.